Filters/src/extensions.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" standalone="yes"?>
+<extension name="Image Filters" version="3.0">
+    <author>Maksim Khramov</author>
+    <date>2013</date>
```

Comment:
For clarity, it might be nice to say "Based on code extracted from AOI core" in the author field

Also, check the date here...

Reply:
Updated date and description